CHANGELOG.md

```diff
@@ -5,6 +5,76 @@ Documentation of all notable changes to the **evmone** project.
 The format is based on [Keep a Changelog],
 and this project adheres to [Semantic Versioning].
 
+[0.14.0] — unreleased
+
+### Added
+
+- Support for all remaining [Prague] EIPs:
+  - [EIP-4844]: Support for blockchain tests.
```

Comment:
technically, this is a Cancun EIP, even though we caught up with tests to implement a Prague EIP.

Reply:
Good one. I moved it to separate item.